ansible/roles/operator-pipeline/templates/openshift/tasks/create-container-image.yml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-container-image
+spec:
+  params:
+    - name: pyxis_api_key_secret
+      default: pyxis-api-secret
+      description: OpenShift secret name that contains Pyxis API key. Valid only when external Pyxis is used.
+    - name: pyxis_cert_path
+      default: ""
+      description: Path to Pyxis certificates. Valid only when internal Pyxis is used.
+    - name: pyxis_key_path
+      default: ""
+      description: Path to Pyxis key. Valid only when internal Pyxis is used.
+    - name: pyxis_url
+      default: https://catalog.redhat.com/api/containers/
+    - name: isv_pid
+      description: isv_pid of the certification project from Red Hat Connect
+    - name: registry
+      description: certification project registry
+    - name: repository_name
+      description: Repository name assigned to certification project from Red Hat Connect
+    - name: container_digest
+      description: imagestream container_digest
+  workspaces:
+    - name: pyxis-ssl-credentials
+  steps:
+    - name: create-container-image
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      env:
+        - name: PYXIS_CERT_PATH
+          value: $(params.pyxis_cert_path)
+        - name: PYXIS_KEY_PATH
+          value: $(params.pyxis_key_path)
+        - name: PYXIS_URL
```

Comment:
Why is this needed if we already have params variables available in a script?

Reply:
It trims down the \n symbol.